### PR TITLE
changed the font-family to Trattatello of quotes

### PR DESCRIPTION
**Pull request by vivientam**
i hv changed the font-family to Trattatello of quotes. pls check.

### CODE REVIEW

File: app/assets/stylesheets/ourstories.css.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 // Place all the styles related to the ourstories controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.quotes{
+  font-size: 50px;
+  font-family: "Trattatello", Helvetica;
+}
```
Comment (h8rry) [inline app/assets/stylesheets/ourstories.css.scss]: You only need to do this once throughout the whole application

File: app/assets/stylesheets/registrations.css.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.quotes{
+  font-size: 50px;
+  font-family: "Trattatello", Helvetica;
+}
```
Comment (h8rry) [inline app/assets/stylesheets/registrations.css.scss]: Can delete

File: app/assets/stylesheets/sessions.css.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.quotes{
+  font-size: 50px;
+  font-family: "Trattatello", Helvetica;
+}
```
Comment (h8rry) [inline app/assets/stylesheets/sessions.css.scss]: Can delete

File: app/assets/stylesheets/static_pages.css.scss.erb [hunk 4]
```diff
@@ -100,4 +100,9 @@ div.carousel-caption{
     background-color: #333;
     color: white;
   }
+}
+
+.quotes{
+  font-size: 50px;
+  font-family: "Trattatello", Helvetica;
 }
```
Comment (h8rry) [inline app/assets/stylesheets/static_pages.css.scss.erb]: Can delete

File: app/assets/stylesheets/subscribers.css.scss [hunk 5]
```diff
@@ -1,3 +1,7 @@
 // Place all the styles related to the subscribers controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.quotes{
+  font-size: 50px;
+  font-family: "Trattatello", Helvetica;
+}
```
Comment (h8rry) [inline app/assets/stylesheets/subscribers.css.scss]: Can delete

**Comment by vivientam** [comment]
I hv amended as stated and changed the font to Lobster. pls check

File: app/assets/stylesheets/subscribers.css.scss [hunk 6]
```diff
@@ -1,3 +1,3 @@
-// Place all the styles related to the subscribers controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/subscribers.css.scss:1]: You should delete this file if you are not using it

File: app/assets/stylesheets/registrations.css.scss [hunk 7]
```diff
@@ -0,0 +1 @@
+
```
Comment (h8rry) [inline app/assets/stylesheets/registrations.css.scss]: You should delete this file if you are not using it

**Comment by h8rry** [comment]
The rest looks good

**Comment by h8rry** [comment]
Not automatically merge-able. Let's rebase and fix conflicts

File: app/assets/stylesheets/subscribers.css.scss (hunk 6 above)
Comment (vivientam) [inline app/assets/stylesheets/subscribers.css.scss:1]: Deleted! :)

**Pull request merged**